test/Android/UtilSpec.hs

```diff
@@ -0,0 +1,55 @@
+module Android.UtilSpec (
+  spec,
+) where
+
+import Control.Monad (forM_)
+import Data.String.Conversion (toString)
+import Data.Text (Text)
+import Strategy.Android.Util (isDefaultAndroidDevConfig, isDefaultAndroidTestConfig)
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+defaultAndroidDevConfigs :: [Text]
+defaultAndroidDevConfigs =
+  [ "lintClassPath"
+  , "debugRuntimeClasspath"
+  ]
+
+defaultAndroidTestConfigs :: [Text]
+defaultAndroidTestConfigs =
+  [ "testApiDependenciesMetadata"
+  , "testReleaseApiDependenciesMetadata"
+  , "testReleaseCompileOnlyDependenciesMetadata"
+  , "androidTestApiDependenciesMetadata"
+  , "debugAndroidTestImplementationDependenciesMetadata"
+  , "releaseUnitTestImplementationDependenciesMetadata"
+  ]
+
+defaultNonDevAndTestConfigs :: [Text]
+defaultNonDevAndTestConfigs =
+  [ "default"
+  , "androidApis"
+  , "implementation"
+  , "coreLibraryDesugaring"
+  , "releaseRuntimeOnlyDependenciesMetadata"
+  , "archives"
+  ]
+
+spec :: Spec
+spec = do
+  describe "isDefaultAndroidDevConfig" $ do
+    forM_ defaultAndroidDevConfigs $ \candidate -> do
```

Comment:
[nit]: I'm pretty sure you can use `for_` here instead of `forM_`.  I think the "M" in `forM_` means "monad".